sponsors/models.py

```diff
@@ -904,6 +940,9 @@ def benefit_feature_class(self):
         raise NotImplementedError
 
     def get_benefit_feature(self, **kwargs):
+        return self.build_benefit_feature(**kwargs)
+
+    def build_benefit_feature(self, **kwargs):
```

Comment:
This can be confusing for newcomers to this model because this can look like as an unnecessary indirection. I'd suggest to go on with a more Django CBV-like strategy to enable full customization on how to build a benefit feature. We could add a new method such as `get_benefit_feature_kwargs`.

This new method would build the default `kwargs` dict and return it **OR** return an empty dict if no benefit should be created. That way, we could still have the generic logic under the `get_benefit_feature` method as well.